modules/parent-join/src/main/java/org/elasticsearch/join/mapper/ParentIdFieldMapper.java

```diff
@@ -73,7 +74,9 @@ public boolean eagerGlobalOrdinals() {
 
         @Override
         public ValueFetcher valueFetcher(SearchExecutionContext context, String format) {
-            throw new UnsupportedOperationException("Cannot fetch values for internal field [" + typeName() + "].");
+            // Although this is an internal field, we return it in the list of all field types. So we
+            // provide an empty value fetcher here of throwing an error.
```

Comment:
missing word

Reply:
Sorry not sure what you're referring to!

Reply:
I thought you wanted it to read as follows:
> provide an empty value fetcher here **instead** of throwing an error.

Reply:
Fixed in a follow-up commit, thanks!